src/Controller/EnMarche/MembershipController.php

```diff
@@ -107,6 +123,25 @@ public function joinAction(Request $request, EntityManagerInterface $manager): R
         ]);
     }
 
+    /**
+     * Send an email to validate email
+     *
+     * @Route("/valider-email", name="app_membership_email_validation")
+     * @Security("is_granted('ROLE_USER')")
+     */
+    public function sendEmailValidationAction(Request $request): Response
+    {
+        $this->addFlash('info', 'Un email de validation à bien été envoyé');
```

Comment:
Can you add this flash message after the `sendMailValidation` ? what happens if an error occur during the send?

Reply:
Typo: `à` > `a`